Client/game_sa/CHandlingEntrySA.h

```diff
@@ -95,111 +96,105 @@ class CHandlingEntrySA : public CHandlingEntry
-    void SetMaxVelocity(float fVelocity) { m_Handling.Transmission.fMaxVelocity = fVelocity; };
+    void SetCarDriveType(eDriveType Type) { m_Handling.Transmission.ucDriveType = Type; }
+    void SetCarEngineType(eEngineType Type) { m_Handling.Transmission.ucEngineType = Type; }
+    void SetNumberOfGears(unsigned char ucNumber) { m_Handling.Transmission.ucNumberOfGears = ucNumber; }
 
-    void SetBrakeDeceleration(float fDeceleration) { m_Handling.fBrakeDeceleration = fDeceleration; };
-    void SetBrakeBias(float fBias) { m_Handling.fBrakeBias = fBias; };
-    void SetABS(bool bABS) { m_Handling.bABS = bABS; };
+    void SetEngineAcceleration(float fAcceleration) { m_Handling.Transmission.fEngineAcceleration = fAcceleration; }
+    void SetEngineInertia(float fInertia) { m_Handling.Transmission.fEngineInertia = fInertia; }
+    void SetMaxVelocity(float fVelocity) { m_Handling.Transmission.fMaxVelocity = fVelocity; }
 
-    void SetSteeringLock(float fSteeringLock) { m_Handling.fSteeringLock = fSteeringLock; };
-    void SetTractionLoss(float fTractionLoss) { m_Handling.fTractionLoss = fTractionLoss; };
-    void SetTractionBias(float fTractionBias) { m_Handling.fTractionBias = fTractionBias; };
+    void SetBrakeDeceleration(float fDeceleration) { m_Handling.fBrakeDeceleration = fDeceleration; }
+    void SetBrakeBias(float fBias) { m_Handling.fBrakeBias = fBias; }
+    void SetABS(bool bABS) { m_Handling.bABS = bABS; }
 
-    void SetSuspensionForceLevel(float fForce);
-    void SetSuspensionDamping(float fDamping);
-    void SetSuspensionHighSpeedDamping(float fDamping);
-    void SetSuspensionUpperLimit(float fUpperLimit);
-    void SetSuspensionLowerLimit(float fLowerLimit);
-    void SetSuspensionFrontRearBias(float fBias);
-    void SetSuspensionAntiDiveMultiplier(float fAntidive);
+    void SetSteeringLock(float fSteeringLock) { m_Handling.fSteeringLock = fSteeringLock; }
+    void SetTractionLoss(float fTractionLoss) { m_Handling.fTractionLoss = fTractionLoss; }
+    void SetTractionBias(float fTractionBias) { m_Handling.fTractionBias = fTractionBias; }
 
-    void SetCollisionDamageMultiplier(float fMultiplier) { m_Handling.fCollisionDamageMultiplier = fMultiplier; };
+    void SetSuspensionForceLevel(float fForce) { m_Handling.fSuspensionForceLevel = fForce; }
+    void SetSuspensionDamping(float fDamping) { m_Handling.fSuspensionDamping = fDamping; }
+    void SetSuspensionHighSpeedDamping(float fDamping) { m_Handling.fSuspensionHighSpdDamping = fDamping; }
+    void SetSuspensionUpperLimit(float fUpperLimit) { m_Handling.fSuspensionUpperLimit = fUpperLimit; }
+    void SetSuspensionLowerLimit(float fLowerLimit) { m_Handling.fSuspensionLowerLimit = fLowerLimit; }
+    void SetSuspensionFrontRearBias(float fBias) { m_Handling.fSuspensionFrontRearBias = fBias; }
+    void SetSuspensionAntiDiveMultiplier(float fAntidive) { m_Handling.fSuspensionAntiDiveMultiplier = fAntidive; }
```

Comment:
Before the PR there was CheckSuspensionChanges call within these methods. Why did you remove it?

Reply:
CheckSuspensionChanges is useless, you can see for yourself

Reply:
@tederis Can I end this conversation?

Reply:
Please, do not mark a conversation resolved if the actual conversastion is in process. CheckSuspensionChanges has a side effect: internally it can call SetSuspensionEnabled. So it's not useless.

Reply:
> Please, do not mark a conversation resolved if the actual conversastion is in process. CheckSuspensionChanges has a side effect: internally it can call SetSuspensionEnabled. So it's not useless.

Ok. Have you looked at SetSuspensionEnabled?

Reply:
Ok. Maybe I was wrong and I will return it with changes.

Reply:
@tederis Can I mark a conversation resolved?

Reply:
Yes, it's resolved, because you did the desired thing (re-add those calls).